src/Apps/Order/Components/ExpressCheckout/index.tsx

```diff
@@ -112,6 +112,7 @@ export const ExpressCheckoutQueryRenderer: React.FC<{ orderID: string }> = ({
       `}
       variables={{ orderID }}
       render={({ props }) => {
+        console.log("****", props)
```

Comment:
whew 😮‍💨

Reply:
I saw it before i submitted the pr and was already removing it - intended to mark it draft but slipped my mind. Sorry.

Reply:
The only way to fix this stuff is to actually resolve these ingrained repeat patterns for good (actions and mindfulness in ones day to day really does matter). What is getting in the way here? Maybe make a list. Try very hard to figure out what that is - and... problem solved.